docs/quickstarts/express.mdx

```diff
@@ -0,0 +1,143 @@
+    {
+      title: "Express Quickstart Repo",
+      link: "https://github.com/clerk/clerk-express-quickstart"
+    }
+  ]}
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "Create a Express application",
+      link: "https://expressjs.com/en/starter/installing.html",
+      icon: "expressjs",
+    }
+  ]}
+>
+  - Install `@clerk/express`
+  - Set your Clerk API keys
+  - Configure Clerk middleware
+  - Protect your routes
+</TutorialHero>
+
+<Steps>
+  ### Install `@clerk/express`
+
+  Add Clerk's Express SDK to your project:
+
+  <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: 'terminal' }}
+    npm install @clerk/express
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    yarn add @clerk/express
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    pnpm add @clerk/express
+    ```
+  </CodeBlockTabs>
+
+  ### Set environment variables
+
+  <SignedIn>
+    Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+  </SignedIn>
+
+  <SignedOut>
+    1. Navigate to the Clerk Dashboard.
+    1. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+    1. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+    1. Paste your keys into your `.env.local` file.
+
+    The final result should resemble the following:
+  </SignedOut>
+
+  ```sh {{ filename: '.env.local' }}
+  CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+
+  This examples uses `dotenv` to load the environment variables. You can use any other library or method, if you so wish.
+
+  <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: 'terminal' }}
+    npm install dotenv
+    npm install -D @types/dotenv
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    yarn add dotenv
+    yarn add -D @types/dotenv
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    pnpm add dotenv
+    pnpm add -D @types/dotenv
+    ```
+  </CodeBlockTabs>
+
+  ### Add middleware to your application
+
+  The `clerkMiddleware` can be registered globally or for specific routes. This examples registers the middleware globally.
+
+  ```ts {{ filename: 'index.ts' }}
+  import 'dotenv/config'
+  import express from 'express'
+  import { clerkMiddleware } from '@clerk/express'
+
+  const app = express()
+
+  app.use(clerkMiddleware())
+
+  app.listen(3000, () => {
+    console.log(`Example app listening at http://localhost:${port}`)
+  })
+  ```
+
+  ### Accessing auth state using `getAuth`
+
+  The [`getAuth`](/docs/references/nextjs/get-auth) helper can be used to access the authentication state of the current request. It can also be used to protect your routes.
```

Comment:
We have to explain all the moving parts here

```suggestion
  The [`getAuth()`](/docs/references/express/overview#get-auth) helper retrieves authentication state from the request object.

  [`clerkClient`](/docs/references/express/overview#clerk-client) is an instance of Clerk's JavaScript Backend SDK, which provides access to Clerk's Backend API resources and low-level authentication utilities for JavaScript environments.

  The following example uses `getAuth()` to retrieve the `userId`, which is used to protect the route and is passed to `clerkClient.users.getUser()` to retrieve the current user's `User` object.
```

Reply:
Updated!